jupyterhub/templates/_helpers.tpl

```diff
@@ -112,31 +111,65 @@
 {{- /*
   jupyterhub.commonLabels:
     Foundation for "jupyterhub.labels".
-    Provides labels: app, release, (chart and heritage).
+
+    Provides old labels:
+      app
+      release
+      chart (omitted for matchLabels)
+      heritage (omitted for matchLabels)
+    Provides modern labels (omitted for matchLabels):
+      app.kubernetes.io/name ("app")
+      app.kubernetes.io/version
+      app.kubernetes.io/instance release ("release")
```

Comment:
```suggestion
      app.kubernetes.io/instance ("release")
```
Is this a typo?

Reply:
Latest changes look good! Just this nitpick/question from me, then this is good to merge.

Reply:
100% typo - thank you for looking carefully at this @manics!